Deploy r-builds job failing

The latest master build failed on the deploy step, which seems to have been caused by a permissions error:
```sh
2021-01-26 14:43:18  aws s3 cp s3://rstudio-serverless/serverless/r-builds/serverless-custom.yml .
2021-01-26 14:43:20  fatal error: An error occurred (403) when calling the HeadObject operation: Forbidden
2021-01-26 14:43:20  Makefile:31: recipe for target 'fetch-serverless-custom-file' failed
```

https://build.rstudioservices.com/blue/organizations/jenkins/hostedapps%2Fr-builds/detail/master/16/pipeline
https://build.rstudioservices.com/job/hostedapps/job/deploy/job/r-builds/36/

(cc @stevenolen)

--- a/Makefile
+++ b/Makefile
@@ -27,8 +27,11 @@ docker-shell-r-env:
 ecr-login:
 	(aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin $(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com)
 
+push-serverless-custom-file:
+	aws s3 cp serverless-custom.yml s3://rstudio-devops/r-builds/serverless-custom.yml
+
 fetch-serverless-custom-file:
-	aws s3 cp s3://rstudio-serverless/serverless/r-builds/serverless-custom.yml .
+	aws s3 cp s3://rstudio-devops/r-builds/serverless-custom.yml .
 
 rebuild-all: deps fetch-serverless-custom-file
 	$(SLS_BINARY) invoke stepf -n rBuilds -d '{"force": true}'

